announcements/move-to-cluster-api.md

```diff
@@ -0,0 +1,7 @@
+Detailed explanation of what Cluster API is and what the migration means to the customers.
+
+### What is Cluster API?
```

Comment:
I was planning to link this to the source of this text: https://github.com/kubernetes-sigs/cluster-api
but I'm not sure about the warning from that page: 

> NB: Cluster API is still in a prototype stage while we get feedback on the API types themselves. All of the code here is to experiment with the API and demo its abilities, in order to drive more technical feedback to the API design. Because of this, all of the codebase is rapidly changing.

If we're not concerned about it, I'll link to the upstream documentation

Reply:
Maybe this is better https://cluster-api.sigs.k8s.io. At the same time, I would add the project was created two years ago, it is rapidly growing and [beta version is coming soon](https://github.com/kubernetes-sigs/cluster-api/discussions/4351)

Reply:
Sounds good, I'll add it, thanks!